### ISSUE TITLE
Question About Loss

### ISSUE THREAD

**Issue by DuanWei1234** [Francis-Rings/StableAnimator#101]
When I train or fine-tune Stable Diffusion, I find that the loss function keeps oscillating and shows no tendency to decrease. Is this normal? Does the loss converge in your training process?

**Comment by Francis-Rings** [comment]
Hi, the speed of coverage is influenced by the scale, diversity, and quality of your training videos. Additionally, please ensure that your training script is correct, particularly in the diffusion loss component.
